cmd/vpc/vpc.go

```diff
@@ -77,38 +77,56 @@ var createVpcCmd = &cobra.Command{
 	Short: "Create a VPC in YugabyteDB Managed",
 	Long:  "Create a VPC in YugabyteDB Managed",
 	Run: func(cmd *cobra.Command, args []string) {
-		// Validations
-		if !cmd.Flags().Changed("global-cidr") && !cmd.Flags().Changed("cidr") {
-			logrus.Fatal("Either global-cidr or cidr must be specified")
-		}
-
-		if len(createRegions) != len(createCidrs) {
-			logrus.Fatal("Number of regions and cidrs must be equal")
-		}
-
 		vpcName, _ := cmd.Flags().GetString("name")
 		cloud, _ := cmd.Flags().GetString("cloud-provider")
 		globalCidrRange, _ := cmd.Flags().GetString("global-cidr")
 
-		// global CIDR only works with GCP
-		if cloud != "GCP" && cmd.Flags().Changed("global-cidr") {
-			logrus.Fatal("global-cidr is only supported for GCP")
+		// If the cloud is Azure and the FF is off, then the customer must NOT specify a CIDR
```

Comment:
I thought a private VPC + CIDR is always required for Azure.

Reply:
A slightly tangential question: Do we throw error for multi-region clusters and read replicas for Azure in the CLI or do we re-throw the error from the backend?

Reply:
IOW, if we rely on backend to throw errors - since it has a feature flag already - how to we reconcile the FF in both CLI and backend?

Reply:
@kc-yb 
> Do we throw error for multi-region clusters and read replicas for Azure in the CLI or do we re-throw the error from the backend?

IIRC, we aren't throwing error from apiserver for Azure multi-region / read-replica. This is on my radar to take care of before the launch, aiming to work on this within this week. Will also start a thread on the channel to get more inputs from everyone, in case I'm missing more features to restrict.

Thanks for raising this point!

Reply:
Perhaps the CLI can query the VISIBLE feature flags like UI and make a decision. But then we are making dependent on the API server and makes it difficult to deprecate feature flags. Brought this up so that we have a consistent way of handling things.